grpc/Cargo.toml

```diff
@@ -26,6 +26,7 @@ serde = { version = "1.0", features = ["derive"] }
 log = "0.4"
 url = "2.3"
 uuid = { version = "1.3", features = ["v4", "fast-rng"] }
+x509-parser = "0.16.0"
```

Comment:
Do we really need to fix event the patch version of would `x509-parser = "0.16"` be better?